src/SFA.DAS.AANHub.Application/Common/Validators/RequestedByMemberId/RequestedByMemberIdValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId
+{
+    public class RequestedByMemberIdValidator : AbstractValidator<IRequestedByMemberId>
+    {
+        public const string RequestedByMemberIdEmptyErrorMessage = "RequestedByMemberId can't be empty";
+        public const string RequestedByMemberIdNotFoundMessage = "RequestedByMemberId was not found";
+        public RequestedByMemberIdValidator(IMembersReadRepository membersReadRepository)
+        {
+            RuleFor(x => x.RequestedByMemberId)
+                .NotNull()
+                .NotEmpty()
```

Comment:
I think you may need something like `.NotEquals(Guid.Empty)` for your empty guid

Reply:
NotEmpty should take care of it, we have a  test case for this which passed I believe.

Reply:
I can confirm the unit test case had passed